### DISCUSSION TITLE
How to immigrate from gitlab to GitHub interprise

### DISCUSSION THREAD

**Discussion by sumeidibr** [community/community#128949] [Enterprise]
### Body

Do you know if is it possible to immigrate from gitlab to GitHub interprise? If yes, what is the step by step to achieve it?

**Comment by Davdadev** [comment]
**GitLab to GitHub.com
Enterprise Cloud Importer is only available for expert-led migrations. If an expert-led migration isn't right for you, you can perform a "source and history" migration of the affected repositories instead. For more information, see "Migrations from any Git repository to GitHub.com."**

**Reply by mickeygousset** [reply]
As @Davdadev mentioned, GitHub has some tools for doing a high-fidelity migration from GitLab to GitHub.  But that does require some sort of services engagement with GitHub to utilize.  The GitHub Enterprise Importer doesn't support GitLab at this time.
